hl7/4.2/modules/ROOT/edits/index.adoc

```diff
@@ -0,0 +1,174 @@
+console tool to generate a simplified schema based on one or more
+sample documents. It is also possible to use an overlay schema for
+HL7, as with other EDI formats, but this option is not recommended
+due to the complexity of HL7 definitions.
+
+You can define your own schemas from scratch, or copy and edit a base HL7 schema definition to suit your data as shown in the xref:x12-edi-connector::x12-edi-schema-language-reference.adoc[EDI Schema Language Reference].
+
+=== Finding the HL7 Schema Location
+
+To use the connector, you need to know the locations of the schemas
+in your project. If you're using the out-of-the-box HL7 schemas and
+not customizing anything, the standard schema location follows the
+`/hl7/{version}/{message structure}.esl` pattern and the lax schema
+location follows the `/hl7lax/{version}/{message structure}.esl` pattern.
+
+For example, if you're using the 2.5.1 version and the ADT_A01 message
+structure, your schema location is `/hl7/v2_5_1/ADT_A01.esl` for the
+standard version or `/hl7lax/v2_5_1/ADT_A01.esl` for the lax version.
+
+If you're using one or more custom schemas, put these under
+a directory in `src/main/mule` and refer to the location using `${app.home}`.
+For example, if you've put your ADT_A01 schema under `src/main/mule/mypartner/ADT_A01.esl,
+your schema location is `${app.home}/mypartner/ADT_A01.esl`.
+
+The Mule runtime engine automatically checks `src/main/mule` for any locations
+that contain the `${app.home}` value.
+
+== The Hierarchy of X12 EDI Connector Message Structure
+
+Use this connector to read or write HL7 documents into (or from) the canonical ER7 message structure.
+This structure is a hierarchy of Java Maps and Lists. You use either DataWeave or code to manipulate them.
+Each transaction structure is defined in the schemas.
+
+The message itself contains the following keys, some of which apply only to either the read operation or the write operation, as indicated:
+
+[%header%autowidth.spread]
+|===
+|Key name |Description
+|ACK (read only) |Message generated by the module during the read operation. The `MSA-01` acknowledgment code value is based on the parser configuration settings.
+|Data (read or write) |Wrapper for message data, with a key matching the message structure ID value that links to the actual data. Allows different messages to be included in the metadata and handled in DataWeave mappings.
+|Delimiters (read or write) |The delimiters used for the message.
+The characters in the string of delimiters are interpreted based on their position in string, in the following order: component separator, repetition separator, escape character, and subcomponent separator, with a value of `U` indicating no value in that position.
+|Errors (read only) |A list of errors associated with the input message.
+|Id |Message structure ID
+|MSH (read only) |Link that receives MSH segment data
+|Name (read only) |Message structure name
```

Comment:
These have to end with periods since their peer descriptions do. 
Try to make peer elements either all sentences or all fragments, to avoid this kind of issue.